azure-pipelines.yml

```diff
@@ -20,7 +20,8 @@ jobs:
         displayName: Cache Yarn packages
 
       - script: |
-          yarn install --frozen-lockfile --check-files
+          yarn install --immutable
+          yarn prepublish
```

Comment:
Are there other CIs you're aware of that would need to be similarly changed? I couldn't run the non-GH CIs on my fork.

Reply:
Interestingly, Azure seems to be ignoring my change; the prepublish doesn't seem to be called:
https://dev.azure.com/nextjs/next.js/_build/results?buildId=20261&view=logs&j=d9ce86fb-87f0-553d-3342-3c6f10bf2a76&t=bcc0fbf4-bb1e-5992-3d84-d6fb6861ffec&l=95